doc/source/parameters/cfd/volume_of_fluid.rst

```diff
@@ -53,10 +53,10 @@ The default values of the VOF parameters are given in the text box below.
 		subsection surface tension force
 			set enable 	= false
 			set verbosity 	= quiet
-			set output auxiliary fields 	   = false
-			set surface tension coefficient    = 0.0
-			set phase fraction gradient filter = 0.5
-			set curvature filter 		   = 0.5	
+			set output auxiliary fields 	          = false
+			set surface tension coefficient           = 0.0
+			set phase fraction gradient filter factor = 4
+			set curvature filter factor 		  = 1
```

Comment:
Out of curiosity, are the new default values for the parameters equivalent to the old ones? Or no, because the point was to use parameters that do not depend on mesh size?

Reply:
There are not equivalent. We used to give the filter value n_filter and it was recommended to select it base on the mesh size, e.g.  n_filter=h/10. Now, n_filter is defined as n_filter=beta*h^2, and only beta is given as the input.